### ISSUE TITLE
Error Reports can be much more specific

### ISSUE THREAD

**Issue by FarzanRNobakht** [garethr/kubeval#56]
I have a yaml file containing configurations for multiple Kubernetes resources

when i run kubeval 

I get  a one line report!:

```* Missing a kind key```

and it does not specify where in file the error has occured

**Comment by garethr** [comment]
Good idea. I'll look into whether the YAML parser exposes this information, or I need to do something when handling errors of that type.

**Comment by wkhere** [comment]
Yep,

adding a line number would be an obvious helpful addition to the error reporting

**Comment by orrc** [comment]
Similarly, indicating _which_ file failed would be useful.  e.g. this runs against 16 files, but aborts if a single file fails, without mentioning which one:
```
$ kubeval -v 1.11.4 --strict **/*.yml
1 error occurred:

* Missing a kind key
```

And yes, line number would definitely be useful, e.g. a situation I just had: 😄 
```
$ kubeval -v 1.11.4 --strict **/*.yml
…
The document dev/foo_deployment.yml contains an invalid Deployment
---> name: Additional property name is not allowed

$ grep name: dev/foo_deployment.yml | wc -l
      23
```

**Comment by salahalsaleh1993** [comment]
Any updates on this?

**Comment by alexizmailov** [comment]
Any updates?

**Comment by uberspot** [comment]
+1 for a --verbose flag.

**Comment by tielur** [comment]
+1 this would be super helpful RIGHT now 😆
